ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/TimeModelImpl.scala

```diff
@@ -21,7 +23,7 @@ import scala.util.Try
   *                              Must be greater than the derived minimum time to live.
   * @throws IllegalArgumentException if the parameters aren't valid
   */
-case class TimeModel private (
+case class TimeModelImpl private (
```

Comment:
Do we really want to call this `TimeModelImpl`? I'd rather like to go away from this naming convention. We can probably remove the trait, considering they are in the same package anyway.

Reply:
I didn't like it either. Lets just merge the code and the trait.

Reply:
👍